clients/csharp-uwp/UWPVoiceAssistantSample/Assets/defaultConfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "SpeechSubscriptionKey": "aaaa1234bbbbccccddddeeee12345678",
+  "AzureRegion": "westus",
+  "CustomSpeechId": "",
+  "CustomVoiceIds": "",
+  "CustomCommandsAppId": "",
+  "BotId": "",
+  "KeywordActivationModel": {
```

Comment:
NOTE: this is a potentially contentious change but it was really weird to have "KeywordActivationModel..." in front of so many top-level fields -- it just screams for an object.